jupyterlab_requirements/cli.py

```diff
@@ -696,7 +696,6 @@ def lock(
         click.echo("python3 kernel name, cannot be overwritten, assigning default jupyterlab-requirements")
 
     click.echo(f"Kernel name is: {results['kernel_name']!s}")
-    click.echo(f"runtime environment used: {results['runtime_environment']}")
```

Comment:
What about checking for the key's presence (`runtime_enviroment`) and show it if present / skipping if not?
(instead of never showing it)

Reply:
I removed it before, the issue is not present if you run the command from notebook, but from the CLI yes. unless there is a reason to expose the runtime environment used, I think we are good.

Reply:
:+1: ok then